service-spec/service-spec-commons/src/main/java/io/hyscale/servicespec/commons/activity/ServiceSpecActivity.java

```diff
@@ -18,7 +18,11 @@
 import io.hyscale.commons.models.Activity;
 
 public enum ServiceSpecActivity implements Activity {
-    FAILED_TO_DESERIALIZE_REPLICAS("Cannot process the field 'replicas' with error {} , deployment continued with min 1 replica");
+    FAILED_TO_DESERIALIZE_REPLICAS("Cannot process the field 'replicas' with error {} , deployment continued with min 1 replica"),
+    SERVICE_NAME_MISMATCH("Service name in the file name did not match with service name given in the service spec."),
+    PROFILE_NAME_MISMATCH("Expecting profile file name convention to be <env-name>-<service-name>.hprof.yaml,But {} does not match with env or service names specified in the profile file"),
```

Comment:
Let the service name mismatch be similar to profile name mismatch.Make the convention visible in the error message.